_includes/scenarios/integrate/updating-identityattributes-of-yourself.md

```diff
@@ -1,17 +1,45 @@
-{% include warnings/feature-work-in-progress %}
+TODO: change title to "Succeeding Attributes to update values" ?
 
-# Flow
+In live it may happen that personal circumstances change, e.g. you change your telephone number.
```

Comment:
"In live" is not something you say.

Also, "personal circumstances change" sounds rather dramatic, like a divorce or an illness. Luckily, attribute successions don't have to deal with topics quite as serious :-) I don't think the concept of changing personal data needs any sort of motivation or justification, it's an absolutely pervasive notion for any kind of software that manages user-related data.

I'd suggest to just get right into it. Maybe something like "The way Enmeshed handles updates to your personal data is by succeeding old attributes with new ones. Instead of simply replacing old data, a wholly new attribute is created which superseeds its predecessor. Depending on the type of attribute, any peers, with whom you've shared the (old) attribute in the past, are notified about the change to the data about you which they hold a copy of."